qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_base.py

```diff
@@ -273,3 +275,140 @@ def error_message(self, mode: Optional[Union[constants.ERRX.Mode,
         msg = MessageBuilder().errx_query(mode=mode)
         response = self.ask(msg.message)
         return response
+    
+
+    def setup_staircase_sweep(
+        self,
+        v_start: float,
+        v_end: float,
+        N_steps: int,
+        post_sweep_voltage_val: int = constants.WMDCV.Post.STOP,        
+        sweep_chan = 1,
+        measure_chan_list = [1],
```

Comment:
It normally a bad idea to use mutable default arguments. I suggest using a tuple here 

https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Thanks! I will keep this in mind. As a side note, I will create a new PR for SMU driver only and abandon this one. The CMU part is already done in #2001.